app/views/data_tables/_data_table.html.haml

```diff
@@ -13,11 +13,10 @@
                                                                 :options => options }
 
 .clearfix
-.dxDataGrid-holder{:id => "#{container_id}-holder"}
+.dxDataGrid-holder{:id => "#{container_id}-holder", :style=> "height: inherit;"}
   - height_style = (height.present?) ? "height: #{height};" : ""
   - width_style = (width.present?) ? "width: #{width};" : ""
-  - height_number = (height.present?) ? height.gsub('px', '') : 'null'
```

Comment:
any idea why the 'null' was there?

Reply:
`if (!(height || width)){`

`("#{height}")`

In the previous implementation, those two statements needed either a `null` value or a `number`

But with the new implementation it will be an empty string `""` if there's no value, so it will still work the same cause `if("")` is still falsey